### ISSUE TITLE
Catch syntax error - dot-prefixed variable usage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#59]
```
What steps will reproduce the problem?
1. echo 'var b = 1; var a = .b;' > test.js
2. gjslint test.js


What is the expected output?

Syntax error should be reported. The `.b` is invalid and causes an error in 
browsers (tested with Chrome 23.0.1271.95)


What do you see instead?

    `1 files checked, no errors found.`



gjslint 2.3.7 on Ubuntu 12.10

```

Original issue reported on code.google.com by `craig.ds@gmail.com` on 3 Dec 2012 at 10:44
